cmake/Findthrift.cmake

```diff
@@ -2,62 +2,62 @@
 
 # - Find Thrift (a cross platform RPC lib/tool)
 # This module defines
-#  THRIFT_VERSION_STRING, version string of ant if found
-#  THRIFT_LIBRARIES, libraries to link
-#  THRIFT_INCLUDE_DIR, where to find THRIFT headers
-#  THRIFT_COMPILER, thrift compiler executable
-#  THRIFT_FOUND, If false, do not try to use ant
+#  thrift_VERSION_STRING, version string of ant if found
```

Comment:
why is this lower-case change needed?

Reply:
fixes warning:

CMake Warning (dev) at /usr/share/cmake/Modules/FindPackageHandleStandardArgs.cmake:272 (message):
  The package name passed to `find_package_handle_standard_args` (THRIFT)
  does not match the name of the calling package (thrift).  This can lead to
  problems in calling code that expects `find_package` result variables
  (e.g., `_FOUND`) to follow a certain pattern.
Call Stack (most recent call first):
  cmake/Findthrift.cmake:92 (FIND_PACKAGE_HANDLE_STANDARD_ARGS)
  CMakeLists.txt:63 (find_package)
This warning is for project developers.  Use -Wno-dev to suppress it.